inc/saiport.h

```diff
@@ -3242,6 +3242,24 @@ typedef enum _sai_port_stat_t
     /** Port stat out drop reasons range end */
     SAI_PORT_STAT_OUT_DROP_REASON_RANGE_END = 0x00002fff,
 
+    /** SAI port stat ether in pkts 1519 to 2500 octets */
```

Comment:
@rajendrat I think it would be better to move this after other SAI stats counters at line 2804